ibm/service/database/data_source_ibm_database_remotes_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package database_test
+
+import (
+	"fmt"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMDatabaseRemotesDataSourceBasic(t *testing.T) {
+	testName := fmt.Sprintf("tf-Pgress-%s", acctest.RandString(16))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheck(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMDatabaseRemotesDataSourceConfigBasic(testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_database_remotes.database_remotes", "deployment_id"),
```

Comment:
Add resource.TestCheckResourceAttrSet to check if leaders and replicas are set

Reply:
@hkantare leaders and replicas are not meant to be set. The only resource that is meant to be set is deployment_id.